RawEventProcessor.cpp

```diff
@@ -460,7 +468,7 @@ bool RawEventProcessor::process_syscall_event(const Event& event) {
     if (path_rec) {
         for (auto &f : path_rec) {
             auto fname = f.FieldName();
-            if (fname != SV_ITEM) {
+            if ((fname != SV_ITEM) && (fname != SV_NODE)) {
```

Comment:
we really need the additional parenthesis around each != clause?

Reply:
I like them but yes, they are unnecessary so I've removed them for you.